code/_helpers/global_lists.dm

```diff
@@ -4,6 +4,7 @@
 var/global/list/cable_list = list()					//Index for all cables, so that powernets don't have to look through the entire world all the time
 var/global/list/landmarks_list = list()				//list of all landmarks created
 var/global/list/side_effects = list()				//list of all medical sideeffects types by thier names |BS12
+var/global/list/atom/teleporter_beacons = list()	// List of all active teleporter beacons in world
```

Comment:
`global` in this context doesn't do anything, the other lists all use it due to cargo cult coding.

Reply:
Can disregard this, apparently it's used by the linter.